### ISSUE TITLE
Conda 4.2 branch doesn't support CIO_TEST.

### ISSUE THREAD

**Issue by mingwandroid** [conda/conda#3203]
In config.py in `conda 4.1.x` I see:

```
def get_channel_urls(platform=None):
    if os.getenv('CIO_TEST'):
        import cio_test
        base_urls = cio_test.base_urls
```

cio_test is not imported anywhere in 4.2.

**Comment by kalefranz** [comment]
I need to discuss with Ilan. Didn't get a chance today.

**Comment by mingwandroid** [comment]
No problem, I've been using `4.1.x` when I need to build those packages anyway.

**Comment by kalefranz** [comment]
resolved with #3395
